clippy_lints/src/matches.rs

```diff
@@ -187,6 +187,25 @@ declare_clippy_lint! {
     "a match on an Option value instead of using `as_ref()` or `as_mut`"
 }
 
+/// **What it does:** Checks for wildcard matches using `_`.
+///
+/// **Why is this bad?** New variants added by library updates can be missed.
+///
+/// **Known problems:** None.
+///
+/// **Example:**
+/// ```rust
+/// match x {
+///     A => {},
+///     _ => {}
+/// }
+/// ```
+declare_clippy_lint! {
+    pub MATCH_WILD,
```

Comment:
```suggestion
    pub WILDCARD_MATCH_ARM,
```
lint names should build a sentence (more or less) if they're put after an "allow": #2845 
"allow wildcard match arm" is more a sentence than "allow match wild".